tests/engine-state/htptopcap.py

```diff
@@ -0,0 +1,128 @@
+#! /usr/bin/env python
```

Comment:
I don't think it makes sense to duplicate this file in our codebase. We should probably focus on merging the libhtp PR and link to that in README or something.
Thoughts, @catenacyber ?

Reply:
Actually, this file is not present in the codebase.

It was just on a Draft PR.
It was mentioned on a comment that the tool isn't needed in the repo.

Although, Victor said it could be added in the qa directory, but Phillipe only did that on a Draft PR.

Reply:
No big thoughts on this... You can avocate for it if you want it...

Reply:
I'd love to see it merged somewhere where we could easily refer with a link to. But not sure what's the best place for that. Ideally, not in a specific test, but I'd rather this not to be a blocker for this work to be merged. (IMHO)

Reply:
In that case, please make sure to mention in the file that author is Philippe 😉

Reply:
I'll do just that.

Thank you 😆.